cmake/modules/RootBuildOptions.cmake

```diff
@@ -442,3 +442,12 @@ if(macos_native)
     message(STATUS "Option 'macos_native' is only for MacOS systems. Ignoring it.")
   endif()
 endif()
+
+# Print message saying with which versions of Python are used to build
+if(pyroot_experimental)
```

Comment:
So here it is fine to use this variable?

Reply:
(see below)